packages/salesforcedx-vscode-lwc/src/commands/forceLightningLwcPreview.ts

```diff
@@ -27,119 +23,33 @@ import { PreviewService } from '../service/previewService';
 import { telemetryService } from '../telemetry';
 import { openBrowser, showError } from './commandUtils';
 import { ForceLightningLwcStartExecutor } from './forceLightningLwcStart';
-
-enum PreviewPlatformType {
```

Comment:
Many of the deleted codes in this file are actually refactored/moved to the new `lwcUtils.ts`